plugin/src/main/java/at/helpch/chatchat/channel/ChannelTypeRegistry.java

```diff
@@ -28,7 +28,7 @@ public void add(final @NotNull String name, final @NotNull Builder<?> builder) {
         if (builders.containsKey(name)) {
             throw new IllegalStateException("Attempted to register duplicate channel type " + name);
         }
-        builders.put(name, builder);
+        builders.put(name.toLowerCase(), builder);
```

Comment:
Won't this result in possible conflicts as we check if builder contains this key but not lowercased before?

Reply:
correct good spot